added sc4drf: a series

LChenGit opened the pull request:
@chunhualiao 
starting to add data from sc4drf (https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
- this commit contains data name starting with a: DRB182, DRB 183
- updated benchmarkList
- updated DRB-metainfo.json

Please let me know if there is any problem

Thank you!

chunhualiao commented:
@LChenGit  This is a very good initial request.  Mostly please add comments to indicate

1. license information
2. who contributed the files (university and student  + advisor name). 
3. And if the source file has data races or not
4.  if there are data races, please indicate the pairs of variables causing data races. One example is :

/* 
   Data Race Pair, *maxPos@179:9:W vs. *maxPos@177:17:R
                   H[index]@173:5:W vs. H[*maxPos]@177:15:W
 */ 

the convention for each pair should be self explanatory to indicate line_number:col_number:R(ead)|W(write). 
Be careful that adding comments on the top will change the line number. 

Please look at a few existing files to see how comments are added for -yes or -no files. 

Please let me know once you have done this.   I will check again.

LChenGit commented:
@chunhualiao  Hi Leo. I have updated the commit following your comments. Please let me if it works. Thank you!

LChenGit commented:
@chunhualiao Hi Leo. The following changes have been made:
1. removed the license for the external data. I will send them an email for appropriate license files.
2. updated the license identifier in two new added DRB files
3. update the name in the benchmarkList with links.
I am checking if there is a good way to update the old DRB files with the license identifier. 
Please let me know if I missed anything. Thank you!
